content/en/docs/instrumentation/net/automatic.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Using instrumentation libraries
+weight: 3
+linkTitle: Libraries
```

Comment:
@cartermp - I'd like to suggest that we use the same title and linkTitle here as the corresponding page in the Java docs, namely:

```yaml
title: Automatic Instrumentation
linkTitle: Automatic
```